src/createMollieClient.ts

```diff
@@ -55,7 +57,7 @@ function createHttpClient(options: MollieOptions): AxiosInstance {
   axiosOptions.headers['User-Agent'] = [
     `Node/${process.version}`,
     `Mollie/${libraryVersion}`,
-    ...(customVersionStrings as Array<string>).map(versionString => {
+    ...(customVersionStrings as string[]).map(versionString => {
```

Comment:
(customVersionStrings as string[]) I think should always be an array so to cast it is a bit weird imo.

Reply:
I disagree. I like that we currently allow providing a single `versionString`, as well as multiple. I do understand your point that it does feel a bit awkward though.